R/utils-json.R

```diff
@@ -1,4 +1,4 @@
-makeVariableDetails <- function(value, variableId, entityId, displayLabel = NULL) {
+makeVariableDetails <- function(value, variableId, entityId, displayLabel = NULL, naToZero = NULL) {
```

Comment:
this is interesting.. im not sure we need to tell the client weve imputed 0s. is there something im not thinking of?

Reply:
the main place i was thinking of is maybe in the complete cases table, or some other annotation on the plot somewhere that says we imputed zeros. For example, in reality variable A has 50 missing values, but now it looks like it has none. That could be confusing if there wasn't a little asterisk or something to indicate that we've imputed 0s for this variable.

Reply:
sure, but even if we go that route, i think the client probably already knows this? i mean its a decision made per variable based on variable metadata, which the client has access to.